TAS_Engine/Convert/Environment_oM/InternalGain.cs

```diff
@@ -52,45 +54,123 @@ public static List<BHE.Gain> ToBHoM(this TBD.InternalGain tbdInternalGain)
             peopleGain.GainType = BHE.GainType.People;
-            peopleGain.GainUnit = BHE.GainUnit.NumberOfPeople;
             peopleGain.RadiantFraction = tbdInternalGain.occupantRadProp;
             peopleGain.ViewCoefficient = tbdInternalGain.occupantViewCoefficient;
-            peopleGain.SensibleGain = tbdInternalGain.personGain; //ToDo - review if this is the best place for this!
+
+            peopleGain.Profile = aProfile;
             occupantGain.GainProperties = peopleGain;
             occupantGain.CustomData = tasData;
-
             gains.Add(occupantGain);
 
+            //Equipment Sens
             BHE.Gain equipGain = new BHE.Gain();
-            equipGain.Name = tbdInternalGain.name;
-
-            BHP.GainPropertiesEquipmentSensible equipmentGain = new BHP.GainPropertiesEquipmentSensible();
-            equipmentGain.GainType = BHE.GainType.Equipment;
-            equipmentGain.GainUnit = BHE.GainUnit.WattsPerSquareMetre;
-            equipmentGain.RadiantFraction = tbdInternalGain.equipmentRadProp;
-            equipmentGain.ViewCoefficient = tbdInternalGain.equipmentViewCoefficient;
-            equipGain.GainProperties = equipmentGain;
+            equipGain.Name = "EqS " + tbdInternalGain.name;
+            equipGain.GainType = BHE.GainType.Equipment;
+            BHP.GainPropertiesEquipmentSensible equipmentSensibleGain = new BHP.GainPropertiesEquipmentSensible();
+            equipmentSensibleGain.GainType = BHE.GainType.Equipment;
+            equipmentSensibleGain.GainUnit = BHE.GainUnit.WattsPerSquareMetre;
+            equipmentSensibleGain.RadiantFraction = tbdInternalGain.equipmentRadProp;
+            equipmentSensibleGain.ViewCoefficient = tbdInternalGain.equipmentViewCoefficient;
+
+            tbdProfile = tbdInternalGain.GetProfile((int)TBD.Profiles.ticESG);
+            aProfile = tbdProfile.ToBHoM(BHEE.ProfileCategory.Gain);
+            equipmentSensibleGain.Profile = aProfile;
+
+            equipGain.GainProperties = equipmentSensibleGain;
             equipGain.CustomData = tasData;
+            gains.Add(equipGain);
+
+            //Equipment Lat
+            equipGain = new BHE.Gain();
+            equipGain.Name = "EqL " + tbdInternalGain.name;
+            equipGain.GainType = BHE.GainType.Equipment;
+            BHP.GainPropertiesEquipmentLatent equipmentLatentGain = new BHP.GainPropertiesEquipmentLatent();
+            equipmentLatentGain.GainType = BHE.GainType.Equipment;
+            equipmentLatentGain.GainUnit = BHE.GainUnit.WattsPerSquareMetre;
+
+            tbdProfile = tbdInternalGain.GetProfile((int)TBD.Profiles.ticELG);
+            aProfile = tbdProfile.ToBHoM(BHEE.ProfileCategory.Gain);
+            equipmentLatentGain.Profile = aProfile;
 
+            equipGain.GainProperties = equipmentLatentGain;
+            equipGain.CustomData = tasData;
             gains.Add(equipGain);
+            occupantGain.CustomData = tasData;
```

Comment:
Can this be put up with the rest of the `occupantGain` stuff please

Reply:
done